frontend/vitest.config.ts

```diff
@@ -2,15 +2,16 @@
 import { installGlobals } from '@remix-run/node';
 
 import react from '@vitejs/plugin-react';
+import envOnly from 'vite-env-only';
 import tsconfigPaths from 'vite-tsconfig-paths';
 import { configDefaults, defineConfig } from 'vitest/config';
 
 // install global node polyfills
 // see: https://remix.run/docs/en/main/other-api/node#polyfills
-installGlobals();
+installGlobals({ nativeFetch: true });
```

Comment:
Does it mean we can remove `undici` as dependency?

Reply:
Not sure. I was going to look into that after migration..

<https://remix.run/docs/en/main/guides/single-fetch#enabling-single-fetch>